MODULE.bazel

```diff
@@ -18,13 +19,12 @@ bazel_dep(name = "bazel_skylib_gazelle_plugin", version = "1.5.0", dev_dependenc
 bazel_dep(name = "stardoc", version = "0.5.6", dev_dependency = True, repo_name = "io_bazel_stardoc")
 bazel_dep(name = "bazel_ci_rules", version = "1.0.0", dev_dependency = True)
 
-# note that this dev_dependency is only needed because bazel_ci_rules
-# depends on it. If you are not using bazel_ci_rules; ideally we'd be able to remove this.
-bazel_dep(name = "rules_cc", version = "0.0.9", dev_dependency = True)
-
 python = use_extension("@rules_python//python/extensions:python.bzl", "python")
-python.toolchain(python_version = "3.9")
-use_repo(python, "python_3_9")
+python.toolchain(python_version = "3.12")
+use_repo(python, "python_3_12")
+
+internal_deps = use_extension("@rules_python//python/private:internal_deps.bzl", "internal_deps")
+use_repo(internal_deps, "rules_python_internal")
```

Comment:
This is very hacky but until rules_python exposes its pystar config via features this is the best we can do..